### ISSUE TITLE
failure to find turbine-discovery:jar

### ISSUE THREAD

**Issue by mliu5715k** [fabric8io/kubeflix#53]
maven can't find the needed snapshot, which repo should I use? can someone share the url? 
```
[ERROR] Failed to execute goal on project turbine-server: Could not resolve dependencies for project io.fabric8.kubeflix:turbine-server:war:1.0.12-SNAPSHOT: Failure to find io.fabric8.kubeflix:turbine-discovery:jar:1.0.12-SNAPSHOT in https://oss.sonatype.org/content/repositories/snapshots was cached in the local repository, resolution will not be reattempted until the update interval of sonatype-nexus-snapshots has elapsed or updates are forced -> [Help 1]
```

**Comment by iocanel** [comment]
@mliu5715k: The released artifacts are available at central: http://repo1.maven.org/maven2/io/fabric8/kubeflix/ I am not sure about snapshots, though (I don't think that we publish snapshots at this point).

Now, given that 1.0.14 is already released you could use this rather than 1.0.12-SNAPSHOT.

**Comment by mliu5715k** [comment]
@iocanel: thanks i replaced the version and am now getting different error, please see the new issue i submitted with regard to setup dependencies.
```diff pom.xml pom.xml.bak
24c24
<         <version>1.0.14</version>
---
>         <version>1.0.12-SNAPSHOT</version>```
